qrencode.el

```diff
@@ -1084,6 +1084,15 @@ Commands:
         (message "No URL found at point")
       (qrencode--encode-to-buffer url))))
 
+;;;###autoload
+(defun qrencode-url (url)
+  "Encode URL.
+When called interacively, read URL with `read-string'."
+  (interactive `(,(read-string "URL: ")))
+  (if (null (string-trim url))
+      (user-error "No URL entered")
+    (qrencode--encode-to-buffer url)))
```

Comment:
It would probably be good to pass the string-trim version here as well, as this would otherwise encode the spaces as well.

Reply:
Done.